README.md

```diff
@@ -395,7 +396,7 @@ ansible_ssh_private_key_file=/root/.ssh/dev-server/id_vm_rsa
 **Install Kubernetes**
 
 ```
-ansible-playbook -i inventory/vms.local.generated playbooks/kube-install.yml
+ansible-playbook -i inventory/vms.local.generated playbooks/kube-install-ovn.yml
```

Comment:
This `README.md` is general one, so I suppose `kube-install.yml` could be best.
(I hope ovn getting the de-facto standard, however now it isn't...)

Reply:
This is an example section, that actually specifically says that it's going to install kubernetes with ovn, that's the reason i need to change it for correctness purpose. But if we want to show the general case, we can change this example to general example (probably kubernets + multius + flannel ).?

Reply:
Okey, gotcha. it makes sense so let me merge that.